### ISSUE TITLE
Mapped Read File == resquiggle?

### ISSUE THREAD

**Issue by pabloacera** [nanoporetech/taiyaki#98]
Hi,

Thanks for the useful tool.
I was wondering if prepare_mapped_reads.py script does a similar thing as tombo resquiggle does. 
Is it assigning chunks of signals to nucleotides? 
please, How different are these methods?
Thanks a lot in advance for your time.

**Comment by marcus1487** [comment]
Yes these methods are quite similar, but with different goals. The result of each is assigning the raw nanopore signal to reference bases. The Tombo mappings use a k-mer model for expected signal level at each reference base, while the `prepare_mapped_reads.py` method uses a previously trained basecalling model to assign signal to reference bases based on the intermediate basecalling output.

The other major difference is the output format. Tombo is intended to extract signal levels at random from a reference position, while the Taiyaki mapped signal format is designed to extract "chunks" of signal and matching sequence for basecaller training. Either method could in theory be converted to output the other format.
